features/step_definitions/json_api_average.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 Given(/^a set of ratings for one song$/) do
-  6.times { |_i| create :rating, ratable_type: 'jukebox_song', ratable_id: 11 }
+  6.times { |i| create :rating, ratable_type: 'jukebox_song', ratable_id: 11 }
```

Comment:
You should be able to drop the param entirely if you aren't using it. Rubocop will carp about this.